soccer/Processor.cpp

```diff
@@ -402,6 +401,7 @@ void Processor::run() {
         for (NewRefereePacket* packet : refereePackets) {
             SSL_Referee* log = _state.logFrame->add_raw_refbox();
             log->CopyFrom(packet->wrapper);
+            curStatus.lastRefereeTime = packet->receivedTime;
```

Comment:
Just checking, the refereePackets list is sorted? If not we should put this in a max along with the old lastRefTime